mobile_verifier/src/cli/server.rs

```diff
@@ -143,22 +166,40 @@ impl Cmd {
             mobile_rewards,
             reward_manifests,
             price_tracker,
-            data_transfer_ingest,
         );
 
+        // TODO: retrieve initial carrier keys from config service
+        let subscriber_location_ingestor = SubscriberLocationIngestor::from_settings(
+            settings,
+            pool.clone(),
+            config_client.clone(),
+            Vec::new(),
+            subscriber_location_ingest,
+        );
+
+        let data_session_ingestor = DataSessionIngestor { pool: pool.clone() };
```

Comment:
this should probably be created from a `new` or `from_settings` function as well (new seems more appropriate here since it's just the one field)

Reply:
addressed here: 97084c89f45b29e241c4d7c6c2bc2c25c694197f